simtools/data_model/format_checkers.py

```diff
@@ -21,6 +21,20 @@ def check_astropy_unit(unit_string):
     return True
 
 
+@format_checker.checks("astropy_unit_of_time", raises=ValueError)
```

Comment:
In case it is not convertible you will get `astropy.units.core.UnitConversionError` rather than a `ValueError`.

Reply:
I have removed the `raises=...` in all decorators. We are not using them and it is fine to let the code raises whatever Exception is appropriate.

Reply:
OK!